.github/workflows/build-artifacts.yml

```diff
@@ -30,14 +30,14 @@ jobs:
       uses: actions/upload-artifact@v2
       with:
         name: jetpack-dev
-        path: /tmp/artifact
+        path: /tmp/artifact/jetpack-dev/projects/plugins/jetpack
```

Comment:
I think the difference is here. On L28, it builds into a `jetpack-dev` folder, but the path we are using here is directly into the plugin vs the level above it?

Reply:
Hmm. Will the Automattic/action-jetpack-build-to-branch bit above also need updating somehow or other? ... Looks like yes, but only to change https://github.com/Automattic/action-jetpack-build-to-branch/blob/master/entrypoint.sh#L35 to use `tools/prepare-built-branch.sh` rather than `bin/prepare-built-branch.sh`. I have to wonder why we have that logic in a separate repo.